### PR TITLE
Added space between link

### LINKED ISSUE #15232: The padding for query result tool links is not even.
### Describe the bug
A very minor issue: 
The padding for query result tool links is not even.

### To Reproduce

1. Go to https://demo.phpmyadmin.net/master-config/sql.php?db=phpmyadmin&table=pma__bookmark&pos=0
2. See the query result tool links

### Screenshots
chrome_2019-04-29_15-03-58

### Expected behavior

Should have even padding between each link and it's surrounding square brackets:
chrome_2019-04-29_15-13-50

### PR DESCRIPTION
**Pull request by yashrajbothra**
Signed-off-by: Yash Bothra <yashrajbothra786@gmail.com>

### Description

added `&nbsp` between links.

Fixes #15232

### CODE REVIEW

File: libraries/classes/Html/Generator.php [hunk 1]
```diff
@@ -997,13 +997,13 @@ public static function getMessage(
...
             ) {
-                $inline_edit_link = ' ['
+                $inline_edit_link = ' [&nbsp;'
                     . self::linkOrButton(
                         '#',
                         _pgettext('Inline edit query', 'Edit inline'),
                         ['class' => 'inline_edit_sql']
                     )
-                    . ']';
+                    . '&nbsp;]';
```
Comment (williamdes) [inline libraries/classes/Html/Generator.php]: does the codebae use the same method for pascing ?

Comment (yashrajbothra) [reply libraries/classes/Html/Generator.php]: Yes, I copied it from other link
image

Comment (williamdes) [reply libraries/classes/Html/Generator.php]: Okay, thank you for the confirmation

**Review by williamdes — APPROVED** [review]

**Review by MauricioFauth — COMMENTED** [review]
Looks like there is a missing space around the Refresh link as well.

**Review by MauricioFauth — APPROVED** [review]

**Comment by williamdes** [comment]
Thank you @yashrajbothra for the UI improvement !

**Pull request merged**